router/config.go

```diff
@@ -110,6 +110,14 @@ type ConfigHTTP struct {
 	MaxIdleConnections    int
 	ResponseHeaderTimeout int
 	RedirectHTTP          bool
+	AutomaticGzip         automaticGzip `json:"AutomaticGzip"`
```

Comment:
I would avoid specifying the JSON names, for consistency sake. If we just leave it all upper-case, then it's all uniform.